demo/src/lib/stackblitz/index.ts

```diff
@@ -5,21 +5,23 @@ import type {AsyncFilesSet, SampleInfo} from '../layout/sample';
 
 type StackblitzProcessor = (project: Project, sample: SampleInfo, framework: Frameworks) => Promise<void>;
 
+const SAMPLES_COMMON_ASSETS = /@agnos-ui\/common\/samples\/[^/]+/g;
+
+const cleanUpFile = (content: string) => {
+	return content.replace(SAMPLES_COMMON_ASSETS, '.');
```

Comment:
It would be better to do this during the build.
Note that this only applies when opening the sample in stackblitz, whereas it would be good to also see the simple local relative path ('./...' without '@agnos-ui/common/samples') when showing the source code in the demo site.